### DISCUSSION TITLE
Process for approving/merging PRs?

### DISCUSSION THREAD

**Discussion by jeffmartens** [portland-startups/docs#97] [General]
With a large community contributing, there have already been examples of people submitting competing updates (e.g. one person says a company is out of business, another says its still in business). Do we want to consider a PR review process?

**Comment by TravisCannon** [comment]
Including a link to a news article, blog post, or other relevant evidence in the pull request description might be a good idea.

Some startups do disappear quietly. In those cases, perhaps a +1 from someone else besides the submitter or confirmation from the founders might be good

**Comment by turoczy** [comment]
Thanks for suggesting this, @jeffmartens! 

I like the "citation needed" thinking, @TravisCannon.

At this point, I welcome encountering this sort of conflict, because that means folks are actually engaging with the resource 😂

**Reply by turoczy** [reply]
Example #98
